server/src/test/java/org/jboss/as/controller/persistence/GitPersistenceResourceTestCase.java

```diff
@@ -55,7 +61,7 @@ public void deleteDirectoriesAndFiles() throws Exception {
         if (repository != null) {
             repository.close();
         }
-        FileUtils.delete(root.toFile(), FileUtils.RECURSIVE | FileUtils.RETRY);
+        FileUtils.delete(root.getParent().toFile(), FileUtils.RECURSIVE | FileUtils.RETRY);
```

Comment:
This starts to fail on Windows with something like

```
java.io.IOException: Could not delete file C:\j\asts-core\7c460853\server\target\local6871736874802616330\standalone\.git\objects\4a\3331fe1c18a3f5fb27aab9c7c49d1aa44b4d3a
	at org.eclipse.jgit.util.FileUtils.delete(FileUtils.java:214)
	at org.eclipse.jgit.util.FileUtils.delete(FileUtils.java:164)
	at org.eclipse.jgit.util.FileUtils.delete(FileUtils.java:166)
	at org.eclipse.jgit.util.FileUtils.delete(FileUtils.java:166)
	at org.eclipse.jgit.util.FileUtils.delete(FileUtils.java:166)
	at org.eclipse.jgit.util.FileUtils.delete(FileUtils.java:166)
	at org.jboss.as.controller.persistence.GitPersistenceResourceTestCase.deleteDirectoriesAndFiles(GitPersistenceResourceTestCase.java:64)
...
Caused by: java.nio.file.AccessDeniedException: C:\j\asts-core\7c460853\server\target\local6871736874802616330\standalone\.git\objects\4a\3331fe1c18a3f5fb27aab9c7c49d1aa44b4d3a
...
```

I'm not sure how much this is an environment issue.

Reply:
With the most recent update to #4095 , the Windows integration tests are successful.